### ISSUE TITLE
webpack CSS source map line numbers in the error messages

### ISSUE THREAD

**Issue by jpokrzyk** [vuejs/vue-loader#190]
CSS line numbers are incorrect in linters (stylint) and preprocessor error messages (webpack, stylus..)

Check out the line numbers. 
![image](https://cloud.githubusercontent.com/assets/4333958/14035730/e5b54c3c-f207-11e5-9b87-30f3764c8502.png)

Per Evan the source map line numbers need to be hacked. Or maybe somehow pad some blank lines.

I'm pretty busy over the next week. But if no one gets to it by then I'll probably take a stab at this.

Originally submitted here: https://github.com/vuejs-templates/webpack/issues/56

**Comment by indirectlylit** [comment]
I'm noticing similar behavior. e.g. this .vue file:

```html
<template>
  test
</template>

<script></script>

<style lang="sass">
## invalid
</style>
```

generates this error:

```
    ERROR in ./~/css-loader!./~/vue-loader/lib/style-rewriter.js!./~/sass-loader?indentedSyntax!./~/vue-loader/lib/selector.js?type=style&index=0!./kolibri/plugins/learn/assets/src/learn-main.vue.html
    Module build failed: 
    <template>
    ^
          Invalid CSS after "": expected 1 selector or at-rule, was "## invalid {}"
          in /Users/d/Projects/learning_equality/repos/kolibri/kolibri/plugins/learn/assets/src/learn-main.vue.html (line 1, column 1)
     @ ./~/vue-style-loader!./~/css-loader!./~/vue-loader/lib/style-rewriter.js!./~/sass-loader?indentedSyntax!./~/vue-loader/lib/selector.js?type=style&index=0!./kolibri/plugins/learn/assets/src/learn-main.vue.html 4:14-303
```

Here it's even more confusing because it actually quotes text from the .vue file, taken from the wrong location, e.g.:

```
    <template>
    ^
```

**Comment by indirectlylit** [comment]
@jpokrzyk - curious if that change helped with your issue?

**Comment by Hucy** [comment]
): It does not seem to work
```
 "vue-html-loader": "^1.0.0",
    "vue-loader": "^8.5.2",
    "vue-style-loader": "^1.0.0",
```
and error:
```
ERROR in ./~/css-loader?sourceMap!./~/vue-loader/lib/style-rewriter.js!./~/stylu
s-loader!./~/vue-loader/lib/selector.js?type=style&index=0!./src/components/News
View.vue
Module build failed: TypeError: e:\vue-hackernews\src\components\NewsView.vue:10
2:1
    98|     a
    99|       margin-right 10px
   100|       &:hover
   101|         text-decoration underline
   102|
--------^
```

**Comment by kazupon** [comment]
Close (fixed #230)
